tests/fxl-properties-001/OPS/package.opf

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout Properties</dc:coverage>
```

Comment:
Maybe just Fixed Layout? Depends on how big we want the test sections to be.

Once decided, please update the running list of coverage sections at https://github.com/w3c/epub-tests#metadata (in this PR).